### ISSUE TITLE
Fetch user data

### ISSUE THREAD

**Issue by Ross-Alexandra** [Ross-Alexandra/SENG-371-Project-2#35]
Blocked by GH-34

Once the system has navigated to the user's data-store, it must pull the data for later processing. This ticket represents the work required for the system to pull and temporarily store the data.

### Acceptance Criteria
 * The system can pull data from a data-store that has been navigated to.

**Comment by joshmcintosh** [comment]
When you mean temporarily store the data, do you mean as in #31 or will this be addressed in another ticket?

**Comment by Ross-Alexandra** [comment]
Temporarily within the container, waiting for the data to be processed. IE, we will not permanently host the unprocessed data, just hold onto it long enough to process it.

**Comment by Ross-Alexandra** [comment]
Completed as part of the final push.
